### PR TITLE
beecrypt, gptfdisk, mapnik: up revision for icu4c update

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

**Comment by DomT4** [comment]
`couchdb` was done a couple days back. Doesn't need redoing.

**Comment by DomT4** [comment]
Ideally, add tests so we can actually catch these things automatically in future. That'd be awesome. I've (attempted) to deal with `mpd` at least in #39360. `couchdb` doesn't seem to have a practical, workable test which is frustrating.

**Comment by vszakats** [comment]
Added strict audit updates and test for `beecrypt`. I'm passing on `mapnik` for now, it's just too large to build and install.

**Comment by jacknagel** [comment]
For future reference, when a formula is updated and its shared library version changes, we need to start from the assumption that *all* dependents need their revisions bumped. If they have dynamic linkage, they will be broken otherwise.

File: Library/Formula/beecrypt.rb [hunk 1]
```diff
@@ -26,8 +24,34 @@ def install
...
+        sha256Process(&hash);
+        sha256Digest(&hash, digest);
+
+        printf("%s\\n", b64crc(digest, 32));
+
+        return 0;
+      }
+    EOS
+    system ENV.cc, "test.c", "-lbeecrypt", "-o", "test"
+    assert_match /ZF8D/, shell_output("./test")
```
Comment (tdsmith) [inline Library/Formula/beecrypt.rb]: Awesome test, thanks! :100:

**Comment by tdsmith** [comment]
Thanks for these; one commit per formula would be great going forward.

**Pull request closed without merging**